docker-compose.yml

```diff
@@ -244,6 +245,10 @@ services:
             - JVB_WS_DOMAIN
             - JVB_WS_SERVER_ID
             - PUBLIC_URL
+            - JVB_NAT_HARVESTER_LOCAL_ADDRESS
```

Comment:
Prefix these 2 with octo, since the names suggest they could be used for regular communication.

Reply:
I'm thinking of using `JVB_OCTO_BIND_ADDRESS` and `JVB_OCTO_PUBLIC_ADDRESS` instead. WDYT?

Reply:
Sounds good!